twistlock/datadog_checks/twistlock/config.py

```diff
@@ -0,0 +1,20 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+
+class Config:
```

Comment:
Since this will be for A6+ only, you can just make all these attributes of `TwistlockCheck`

Reply:
I kinda like the standalone class, makes it a bit cleaner to see what config options there are

Reply:
I did make config an attribute